src/app/components/SalesButton/index.tsx

```diff
@@ -64,7 +64,7 @@ export default function SalesButton({
   return (
     <StyledButton onClick={() => onClick()} disabled={disabled}>
       {text}
-      {loading && <Spinner size={18} className="tw-ml-2" />}
+      {loading && <Spinner size={20} className="tw-ml-2" />}
```

Comment:
Could you please use the enum instead of the enum.

edit: looking at the enum could you also adjust the name to SpinnerSize and the sizes to SM, MD, LG, to fit the usual pattern